src/main/java/org/wiremock/RandomHelper.java

```diff
@@ -0,0 +1,26 @@
+package org.wiremock;
+
+import com.github.jknack.handlebars.Options;
+import com.github.tomakehurst.wiremock.extension.responsetemplating.helpers.HandlebarsHelper;
+import net.datafaker.Faker;
+
+/*
+ * Author: Shreya Agarwal
+ */
+public class RandomHelper extends HandlebarsHelper<Object> {
+
+  @Override
+  public Object apply(Object context, Options options) {
+    Faker faker = new Faker();
```

Comment:
Could this be instantiated once and shared as a field?

Reply:
Converted to instance variable.